doc/source/installation.rst

```diff
@@ -26,6 +26,79 @@ In order to install and run an instance of ``pyramid_oereb``, the following requ
     <https://www.sqlalchemy.org/>`__ and `GeoAlchemy 2 <https://geoalchemy-2.readthedocs.io/en/latest/>`__.
 
+3. **A Service to produce PDF extracts:**
+    This might be a externally running instance of `oereb_xml_pdf_service <https://github.com/Geocloud-AG/oereb_xml_pdf_service>`__
+    (needs registration). Or you can use a self-hosted instance of `mapfish-print <https://github.com/openoereb/pyramid_oereb_mfp>`__
+    prepared for the ÖREB extract.
+
+To straighten this guide we will run them inside a docker composition. This puts aside local differences
+like OS (Mac, Windows, Linux) and safes you from conflicts of maybe already running and conflicting services
+from your other projects
+
+.. _installation-docker-docker-compose:
+
+Docker/Docker-Compose
+---------------------
+
+.. note:: We assume you have a working
+          `docker <https://docs.docker.com/engine/install/>`__/`docker-compose <https://docs.docker.com/compose/install/>`__
+          and `git <https://git-scm.com>`__ setup on your machine.
+
+First we need to get the code. Clone the pyramid_oereb repository in a place of your choice on your machine:
+
+.. code-block:: shell
+
+ git clone https://github.com/openoereb/pyramid_oereb.git
+
+CD into the created directory:
+
+.. code-block:: shell
+
+ cd pyramid_oereb
+
+Docker separates runtimes in so called containers. What is inside a container is defined in an image. These
+images are described by so called Dockerfiles. This repository contains an ``Dockerfile`` in the project root
+in case you want to have a look. A container can be considered existing only at runtime. Once
+it was stopped, all things inside are gone. Each service (pyramid_oereb, database, print-server) will be
+run in its own container.
+
+As we learned above ``pyramid_oereb`` needs different services. The organisation of these services can be done
+by docker-compose. The description of the composition is persisted in a ``docker-compose.yml`` file.
+
+Additionally a docker container can utilize so called ``volumes`` to persist data from inside the container
+over restarts. This is useful especially for configuration but also for data you do not want to be gone after
+containers were stopped. Think of the data you insert into you database or all the python packages you install
+from the web. Wouldn't it be nice to keep them even your containers are stopped? Well... exactly this is how
+this repositories docker composition is set up.
+
+The following steps are required to get a running setup:
+
+#. build docker images locally
+#. setup pyramid_oereb (make build)
+    #. setup all necessary python packages
+    #. create database structure as SQL
+    #. create configuration (wsgi => development.ini)
+    #. create configuration (pyramid_oereb => pyramid_oereb.yml)
+
+Building the docker images locally can be achieved by:
+
+.. code-block:: shell
+
+ # on Linux
+ docker-compose build
```

Comment:
Since we have updated the [gitlab Readme](https://github.com/openoereb/pyramid_oereb/blob/master/README.rst) to use docker compose instead of docker-compose, I think we should stay consistent on this one.

Reply:
Hey Anne. Thank you for checking out. This PR is meant to fix the structure to be able to build the DOC again. The content will be fixed afterwards.